Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -462,6 +471,16 @@ public SortDirection CurrentSortDirection
         /// </summary>
         [Parameter] public bool PreventRowClick { get; set; }
 
+        /// <summary>
+        /// Gets or sets the order for sorting when Sorting is set to multiple. 
+        /// </summary>
+        [Parameter] public int MultipleSortOrder { get; set; }

```

Comment:
I think MultipleSortIndex is a better naming convention. What do you think?

Reply:
Also, does it makes sense to remove `Multiple` from the name?

Reply:
Already have `DisplayOrder` `EditOrder`, makes sense to maintain same consistency.

> Also, does it makes sense to remove Multiple from the name?


Wouldn't suprise me, we can go ahead and remove the Multiple. Just leave the comment mentioning the order only makes sense when it is multiple.

Reply:
I forgot about those. Ok Order can stay but Multiple can be removed.